jobs/uaa/templates/bin/pre-start.erb

```diff
@@ -131,9 +131,16 @@ function process_certs {
 }
 
 function insert_ssl_cert {
+    local FIPS_OPTS=""
+    if [ -f "/proc/sys/crypto/fips_enabled" ]; then
+        local FIPS_ENABLED="$(cat /proc/sys/crypto/fips_enabled)"
+        if [ "${FIPS_ENABLED}" = 1 ]; then
+            FIPS_OPTS="-certpbe PBE-SHA1-3DES"
+        fi
+    fi
     log "Installing Server SSL certificate"
 
-    openssl pkcs12 -export -name uaa_ssl_cert \
+    openssl pkcs12 -export "${FIPS_OPTS}" -name uaa_ssl_cert \
```

Comment:
I think you need to deceide, whether you want "" here our in value, e.g.

FIPS_OPTS=""
+
"${FIPS_OPTS}"

will end up in openssl pkcs12 -export "" -in xxx  but openssl fails then always .
I think change to ${FIPS_OPTS} will fix it

Reply:
done. thx!